hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/RepositoryProperties.java

```diff
@@ -54,6 +54,11 @@
      */
     private boolean eagerPollPersistence;
 
+    /**
+     * If an {@link Action} has a weight of null this value is used as weight.
+     */
+    private int actionWeightIfAbsent = 1000;
```

Comment:
my assumption has been that we want to reduce this value to represent medium priority (e.g. 500) and even make it configurable in a next step...

Reply:
updated it to 500 and also updated relevant test